zaas-service/src/test/java/org/zowe/apiml/acceptance/ZaasTest.java

```diff
@@ -65,4 +86,34 @@ void givenZosmfCookieAndDummyAuthProvider_whenZoweJwtRequest_thenUnavailable() {
         //@formatter:on
     }
 
+    @Test
+    void givenNoCert_whenCallingSafCheck_then401() {
+
+        //@formatter:off
+        given().config(config().sslConfig(new SSLConfig().sslSocketFactory(
+                new SSLSocketFactory(httpConfig.getSecureSslContextWithoutKeystore(), ALLOW_ALL_HOSTNAME_VERIFIER)))
+            )
+            .contentType(APPLICATION_JSON)
+            .body(BODY)
+            .when()
+            .post(String.format("https://%s:%d/zaas/auth/check", hostname, port))
+            .then()
+            .statusCode(SC_UNAUTHORIZED);
```

Comment:
apply formatting first, I think you don't need to provide an SSL config in this scenario
```suggestion
        given()
            .contentType(APPLICATION_JSON)
            .body(BODY)
        .when()
            .post(String.format("https://%s:%d/zaas/auth/check", hostname, port))
        .then()
            .statusCode(SC_UNAUTHORIZED);
```

Reply:
I need it cause otherwise it will fail with
```

PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

Reply:
I can use
`config(SslContext.tlsWithoutCert)`

Reply:
Yes, there's actually another test in this same class that was using the whole thing that can be updated too